eng/vscode/package.json

```diff
@@ -127,6 +127,11 @@
                     "type": "boolean",
                     "default": false,
                     "markdownDescription": "Start the Azure MCP server in read-only mode (operations that modify resources will be disabled)."
+                },
+                "azureMcp.useNonTrimmedBinaries": {
+                    "type": "boolean",
+                    "default": false,
+                    "markdownDescription": "Use non-trimmed Azure MCP server binaries from the `server` folder instead of the default trimmed `server/native` folder. Enable this to opt-out of trimmed binaries for compatibility or debugging purposes."
```

Comment:
@anuchandy should we tweak this description in any way ?

Reply:
I’m wondering if VSIX users even have visibility into the different folders that contain azmcp and its dependencies. If not, maybe we should adjust the wording slightly. For example:

“Opt out of the trimmed single-file Azure MCP Server and use the standard self-contained version (with the .NET runtime, entry point executable, and all DLLs).”